.travis.yml

```diff
@@ -60,4 +60,5 @@ deploy:
   #   on:
   #     tags: true
   #     condition: '$DEPLOY_PYPI == true'
```

Comment:
I added this commented out section in case we ever want Travis to deploy to PyPI. What do you think? It only deploys if it finds a new tag.

Reply:
I would like to make all our codes work this way and this one is the trailblazer.